cmd/createorupdate/createorupdate.go

```diff
@@ -128,7 +129,12 @@ func createOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 		return future.WaitForCompletionRef(ctx, deployments.Client())
 	}
 
-	err = p.CreateOrUpdate(ctx, cs, azuredeploy, oldCs != nil, deployer)
+	var azuretemplate map[string]interface{}
```

Comment:
can we have GenerateARM return map[string]interface{} ?

Reply:
sure, i can give that a go